### PR TITLE
Add DS18B20 sensor package to Makecode Extensions

### PR DESCRIPTION
**Pull request by nielsswinkels**
Added another DS18B20 sensor package for makecode

### Resource description

Another DS18B20 sensor package for makecode. This one supports multiple sensors. Someone else confirmed that it works, even though it is alpha at the moment.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

- [x] I have read and understood the [Contribution Guidelines](https://github.com/carlosperate/awesome-microbit/blob/master/contributing.md)
- [x] I am making an individual pull request for each suggestion
- [x] I have used the correct spelling and capitalisation for `BBC micro:bit`, `micro:bit`, and `Micro:bit Educational Foundation` (resource title could be excluded if there is a good reason)
- [x] The content linked is in English, or it contains an English translation
- [x] I have added the new entry to the bottom of its the category
- [x] I have used the following format:
```
- [Resource Title](link) - Resource short Description (2 lines or less in total)
```

### CODE REVIEW

**Comment by nielsswinkels** [comment]
I don't get what goes wrong with the checking of links? As far as I can see the link just works?

**Comment by carlosperate** [comment]
Right now the main issue is with the O'Reilly server returning an 503 for a different link, they had some server issues due to the fires in California a couple of days ago, this might be related.
Don't worry I will still merge the PR when ready if the CI error is for a different link.

File: README.md [hunk 1]
```diff
@@ -170,6 +170,7 @@ The following extensions can be added into MakeCode by copying the GitHub URL an
 - [ESP-01](https://github.com/51bit/esp01) - Control an ESP8266 module via serial AT commands.
 - [TCS3200](https://github.com/DoraLC/pxt-tcs3200-color-sensor) - Control a TCS3200 color sensor.
 - [IR](https://github.com/lioujj/pxt-IR) - Control IR (infrared) transmitter/receiver modules.
+- [DSTemp Alpha](https://github.com/bsiever/microbit-dstemp-alpha) - Read temperature from DS18B20 sensor. Supports multiple sensors. - Alpha
```
Comment (carlosperate) [inline README.md]:
```suggestion
- [DSTemp Alpha](https://github.com/bsiever/microbit-dstemp-alpha) - MakeCode extension to read temperature from DS18B20 sensor, supporting multiple sensors (alpha release).
```

Comment (carlosperate) [reply README.md]: This change is no longer necessary as for the tweet the section will me automatically included.

**Comment by carlosperate** [comment]
Thanks @nielsswinkels!

**Pull request merged**